packages/terra-site/webpack.prod.config.js

```diff
@@ -2,25 +2,13 @@
 // not devDependencies. Disabling this rule in webpack.conig.js
 /* eslint-disable import/no-extraneous-dependencies */
 
-const webpack = require('webpack');
 const config = require('./webpack.config');
 const CleanPlugin = require('clean-webpack-plugin');
 const path = require('path');
 
-// https://facebook.github.io/react/docs/optimizing-performance.html#use-the-production-build
-config.plugins.push(new webpack.DefinePlugin({
-  'process.env': {
-    NODE_ENV: JSON.stringify('production'),
-  },
-}));
-
```

Comment:
Adding the `-p` flag will automatically add this configuration. https://github.com/cerner/terra-core/issues/283#issuecomment-298723814